docs-client/src/containers/App/index.tsx

```diff
@@ -183,11 +185,19 @@ const AppDrawer: React.FunctionComponent<AppDrawerProps> = ({
         <>
           <ListItem button onClick={toggleServicesOpen}>
             <ListItemText disableTypography>
-              <Typography variant="h5">Services</Typography>
+              <Typography variant="h5">Services </Typography>
             </ListItemText>
             {servicesSectionOpen ? <ExpandLess /> : <ExpandMore />}
           </ListItem>
           <Collapse in={servicesSectionOpen} timeout="auto">
+            <ListItem button onClick={() => navigateTo('/overview')}>
+              <ListItemText>
+                <Typography variant="subtitle2">
+                  <code> Overview </code>
+                </Typography>
+              </ListItemText>
+              <InfoOutlined />
```

Comment:
Personally, I feel like the exclamation mark indicates that the page was somehow loaded incorrectly 😅   
What do you think of removing this?

```suggestion
```